src/game/chars/CCharFight.cpp

```diff
@@ -939,8 +939,21 @@ int CChar::OnTakeDamage( int iDmg, CChar * pSrc, DAMAGE_TYPE uType, int iDmgPhys
 						pSrc->Sound(0x1F1);
 						pSrc->Effect(EFFECT_OBJ, ITEMID_FX_CURSE_EFFECT, this, 10, 16);
 					}
-				}
-			}
+                }
+            }
+            // Check if REFLECTPHYSICALDAM will reflect some damage back.
+            // Preventing recurrent reflection with DAMAGE_REACTIVE.
+            if (!(uType & DAMAGE_REACTIVE))
+            {
+                int iReflectPhysical = minimum(GetPropNum(pCCPChar, PROPCH_REFLECTPHYSICALDAM, pBaseCCPChar),250); //Capped to 250
```

Comment:
Could you use std::min? Minimum is a macro and it calls two times GetPropNum, if you see how it's expanded

Reply:
Fixed. Good point!